src/componentUpdater.ts

```diff
@@ -27,3 +28,33 @@ export function extractComponentProps(component: any, props: any, ignoreProps?:
   }
   return {};
 }
+
+export function extractDayProps(props: CalendarProps) {
```

Comment:
@Inbal-Tish 
Weren't there other extract methods for other components? 
Only the Day component has an "extractProps" method?

Reply:
We have many usages for extractComponentProps, should I include the change in this PR? I thought of opening a different one for the rest...

Reply:
Ok, let's merge this and have a separate one..

Reply:
I added another commit to fix `areEqual` function. Can you please take a look?

Reply:
See my comment.